### PR TITLE
Setup and quick quide

### PR DESCRIPTION
**Pull request by tijmenspeek**

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Installing dependencies
+pip install -r requirements.txt
+echo "✅ Requirements installed"
```
Comment (klimdanick) [inline setup.sh]: maybe dependencies is a better name
?

**Review by klimdanick — APPROVED** [review]
only one comment, not a necessary change. 👍

File: setup.sh (hunk 1 above)
Comment (tijmenspeek) [reply setup.sh]: done

**Review by klimdanick — APPROVED** [review]
👍

**Pull request merged**